pypesto/petab/importer.py

```diff
@@ -78,6 +81,74 @@ def from_yaml(yaml_config: Union[dict, str],
             output_folder=output_folder,
             model_name=model_name)
 
+    def check_gradients(
+        self,
+        *args,
+        rtol: float = 1e-2,
+        atol: float = 1e-3,
+        mode: Literal = None,
```

Comment:
```suggestion
        mode: Optional[Literal] = None,
```

Reply:
Can't use plain `Literal` in `Optional`.